src/meta/src/barrier/command.rs

```diff
@@ -945,15 +947,24 @@ impl Command {
 }
 
 impl CommandContext {
-    pub async fn wait_epoch_commit(&self, epoch: HummockEpoch) -> MetaResult<()> {
+    pub async fn wait_epoch_commit(&self) -> MetaResult<()> {
+        let table_id = self.table_ids_to_commit.iter().next().cloned();
+        // try wait epoch on an existing random table id
```

Comment:
```suggestion
        // try wait epoch on an existing random table id,
        // this is okay because it will be deprecated once we enable partial checkpoint
```

...am I understanding correctly?